config/credentials.yml.enc

```diff
@@ -1 +1 @@
\ No newline at end of file
+zEWBo9XA6+1NxG3EGWgH+4PNc5TdxyeReNkvQOu/2FENqjdhkuRUhJh37OT1A/X+RxiutsG1Zswkd6VsKoPIpvAyQ9cBECSpUuTPVSGbwjQCBv6NsjDHzC5Ui/U+TxKWTxC02m4bmH02tQSqp0ebM1xynD3bGUSz4ZBKWkFEk6RAmdeaqx0K1rYE1GIEY3F9dp/RQkj8MUlH3SPRjI+tOhu+x+487iZK/R23bQafVtfWly/PEPMSbO6nwiVzO/RemC4S5WQ2Auq+JADCb8SVXC9Z2fL5yxK3P0j79jvLUei/mLMUke0mtFTIjA6jE5PyVEN2Rj36KMPghj+8aYnR71fuRlHU0DbwK39FpC+jARviNACGJqUWCzQgWNMswHconsf6W+hzuSyo9RTIFByD+/glPisYSY7gk7n1hUGpdb8Ps/2GC2RZzrOeuNg9R2jDZYntq4J3gC39BpesvheRTyNIK3vMAqWZrfcZeLwLH8AN1EZYHRq4tg7MVIpJ9aVS/HV+DPQ2MyJfwcpJkdmrCROdxAZZpXVfhNF5AquDiGhqRBl2MiUS+eVmfoBok+heOJ6XCKgWCpl0zm2RWiOPIggU4vjxiacgMP0ugd49llKCtJ4qfk+fUSLLO6Ty5TEfAN74keFv4ethzwQYaO/9ts0YlFIdztn42CSD822m8E0ARLyRI4wMTeTqCKvSmzc7+7pWDROYg4Cv8uV+o1spYySQnDaNSil1vesbji4iH2BqyypvhwV5DnRGoYPYcgeV2TsY3mgPrcWFRLMNAWv7XgrnIAw+smden01w407CSWy0cOCpqir/liUxPUvVwrQA7YuIJ3tr50wszlpa2i9qijfAp+KSMq5QyVaHlPYNEQYNSM4Y3pyyVAHbcycwwQXyYB65JVSF3SR1HgGBh6wLRIH7N2Ci4lo4jwH/xF0xCpvClwQncAo8WFfD0S18uRHsmvDsW/tK94K6JyYBZxiyVTzXboiCePNkeezAjZplAOwUQPdoymHaJpdVSvWZ/IU8Xg9k+EU1xeIkbhmTlglIzUUbEuxcuVBRGZlyNJV2VI+a9EtQI+rdu0ZxBj8NGoRrlCFq8ZUjsLbq5iat3FbZlHRGyFUcxXSvK38FKk89kiSF5wXaiMhM1WGwxpW/lHXRkjTf84SzLsxr2zxTncSWBlktJ5O4qbzd24jX2aWVaGpvbvAMjWm+jrhzxguViQGBUBSCIkno/VkHNDson5mnhM0al1FyC3s2e/hRAFGj1WK5qpgty7sT6m4QlfDnpRWkj6XISF/ZQGDs5UXd0RnyDAFZ6RZlYXy7HuwPg6NCe07Eu9XpNn6Ndo+lwt4ft8qnN1YV9auFAgeqi4Y6YSmVeZiorq84Ff/twMX+m8jvoP/ltTMoXEChk/XbEsixrFEoK89WCrnFANuq7ot3xb5kNrj1JztYE047yunxcMdYcYq0mtPQts7WrNtCuD1SPZGIsMEtwJ9D2pIyodhwiYCett7rKPgtsnz1XcTe3K4+wbG60EKZWNtyPfQYGP2SgBISyB0+WnfkZR0Xbgj+jPNEN70JovO7gptixBndwomz0bB8acuSGkdnNYjTzxVgJufQEKWRkEF449F1cLQ99HZAo9kjGfNkyceeHxsTXX1+ufIUqUbDmvXiMsc9VrLDu7C5odMxeBh4zi8TOKCinvZPAjuUieelhAGhat0ASg2JX3FggeovOI+jMsS3E8yifFl3yoJdRb52ZqJLuZ8Rst2MJnSQ8SEvCZ2TTdv44rhb2MqezjoOiObonSiurOTLEdpXRBZmyh7YyZtypc2Ctq/FUnQ4RXSBDqoRrl8K0TtWx4PbMhhrSqF+ZIeEbKfIcVa/Tgzmb33dyWIwcV2PY+7ICHfT41wG/kobM08e6kA8rh8U1fDzuu35lW1rntTjQHmZjKiCCg1eSBcIV9PSzHb2wQZZuf1JCSFMK+AqTWaNf/bAPdAM8ovgael/nBRM9QtSyJT/T5WWhf1gJ2ZFjqkKs+Ogt5M/qMWZhYdPi27U/qEN4E0lCmHHHamTzDNTL5dNKUfbLzr9OQ96LJMV7T2rjLXiueQb/a3ag0bHcjDuEegshDbfcuCbfAGFhYjTPismdgwAQi2eBz7iUReXUsLwiJj0MMlc/g9MTIeYEhIRKq3tAoVKEiludm/lDGrzWSHPuNcT29EgXiOyTkJryonr9YEv1qvI1QyuOno17k8nmlvdl+9gFu6POosB620vn1pT4hgVUZjlgvzb0o60XdMlZiyIZj0uqtJEd9kA7mAjzbyig3F5bSxOG6a5/eRd8HYquBNAntY5oHD3f0Gn3aA0M1d2hekB6HNOiZPNjyvsJSqbAaWpJ89ZWQ5FROjDPGNtuA==--n5r4HJVhqIeP/rVl--ncAp/F0JPY/wUxrb6yVwYw==
```

Comment:
Wie war das hier mit den Credentials? Wieso sind die aktualisiert?
(In der Commitmessage steht dazu nichts.)

Reply:
oben ist ein Foto als Beispiel der Credentials, da diese hier ja verschlüsselt sind

Reply:
aus den werten wird das encrypted, richtig? und weil jetzt mehr werte sind, ist der key auch länger...

Reply:
genau